src/Parser/Monk/Windwalker/CHANGELOG.js

```diff
@@ -1,4 +1,5 @@
 export default `
+16-02-2018 - Added Spinning Crane Kick statistics and suggestion on bad use of Spinning Crane Kick (Juko)
```

Comment:
The string based changelog style is deprecated. Please consider updating the changelog format to the new style when you are able to.